bimmer_connected/cli.py

```diff
@@ -117,7 +118,33 @@ def image(args) -> None:
 def send_poi(args) -> None:
     account = ConnectedDriveAccount(args.username, args.password, get_region_from_name(args.region))
     vehicle = account.get_vehicle(args.vin)
-    poi = PointOfInterest(args.latitude, args.longitude, name=args.name)
+    address = (str(' '.join(args.address)).replace(' ', '+'))
+    url = "https://nominatim.openstreetmap.org/?format=json&addressdetails=1&q="+address+"&format=json&limit=1"
```

Comment:
See https://realpython.com/python-requests/#query-string-parameters. I think this is much easier than +ing all arguments together.

Also imho it would make sense to split this code into a separate function for example `address_to_poi`. That way it could be reused more easily and is easier to test.

Reply:
I really like the idea of a separate function! So instead of calling `bimmerconnected sendpoi ...` maybe we could call `bimmerconnected sendpoi_from_adress ...` (which then of course can use the existing `send_poi` function.

Reply:
Me too. Can you update this? Then we can do some final testing and merge the SendPOI PRs.